packages/test/test-utils/src/testObjectProvider.ts

```diff
@@ -367,7 +367,7 @@ export class TestObjectProvider implements ITestObjectProvider {
             url: await this.driver.createContainerUrl(this.documentId),
             headers: requestHeader,
         });
-        await waitContainerToCatchUp(container);
+        await this.ensureSynchronized();
```

Comment:
I replaced `waitContainerToCatchUp()` to avoid LTS compat issues. Unless we want to explicitly test if the current version of `waitContainerToCatchUp()` can work with older versions of container-loader then I think this should be fine. Is this assumption correct? cc @markfields @vladsud

Reply:
I don't think these are equivalent (they need better names):

- ensureSynchronized waits until amongst all passed containers (or all tracked containers, if none are passed), they are all in "saved" state.  This is generally more applicable to normal runtime behavior, where the containers are sending and receiving ops
- waitContainerToCatchUp waits until the passed container has processed all ops that it knows exist.  This is generally more applicable to the load flow where the container probably hasn't sent anything yet (is "saved") but knows there are more ops available to process still